.github/workflows/release.yml

```
@@ -36,6 +36,25 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
 
+  sonarqube:
```

actionlint (1.7.4)

job "sonarqube" needs job "test" which does not exist in this workflow

(job-needs)